test/wrappers/ortools_test.rb

```diff
@@ -5684,11 +5684,12 @@ def test_force_start_when_there_is_a_service_with_late_tw_begin
     size = 3
 
     # A time matrix that leads to incorrect order in the old force_start implementation
+    array_base = [1]
```

Comment:
This is actually the first column (except the first row). So maybe `array_first_column` ?